site/content/includes/installation/install-gateway-api-resources.md

```diff
@@ -10,6 +10,14 @@ To install the Gateway API resources, run the following:
 kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.0.0/standard-install.yaml
 ```
 
+Alternatively, you can install the Gateway API resources from the experimental channel. We support a subset of the
```

Comment:
I wonder if we need to stay that is not suitable for production use.  According to this https://gateway-api.sigs.k8s.io/concepts/versioning/ , experimental things may introduce breaking changes, but they still can be used in production imho .
Would it make sense to include a link to https://gateway-api.sigs.k8s.io/concepts/versioning/ ?